### PR TITLE
Responsive rubric criteria

### PR DESCRIPTION
**Pull request by valentinludu**
### <samp>🤖 Generated by Copilot at 68903fe</samp>

Improved the UI and UX of the proposal properties and criteria input components by using flexbox and media queries. Refactored some code to use `Box` components for consistency and simplicity.

### WHY
[Task](https://app.charmverse.io/charmverse/page-36967618878533903)

### CODE REVIEW

File: components/proposals/components/ProposalProperties/ProposalProperties.tsx [hunk 1]
```diff
@@ -503,8 +506,21 @@ export function ProposalProperties({
...
+              <Box
+                display='flex'
+                height='fit-content'
+                flex={1}
+                className='octo-propertyrow'
+                flexDirection={{ xs: 'column', sm: 'row' }}
+                alignItems={{ xs: 'stretch!important', sm: 'flex-start' }}
+              >
+                {isMobile ? (
+                  <Typography variant='caption' component='div' color='secondary' fontWeight='600' p='4px 8px' mb={1}>
```
Comment (mattcasey) [inline components/proposals/components/ProposalProperties/ProposalProperties.tsx:518]: Not a big deal but is there a reason u didn't use PropertyLabel?

File: components/proposals/components/ProposalProperties/components/ProposalRubricCriteriaInput.tsx [hunk 2]
```diff
@@ -33,16 +33,22 @@ type Props = {
 export const CriteriaRow = styled(Box)`
   position: relative;
 
-  .show-on-hover {
-    opacity: 0;
-    transform: opacity 0.2s ease-in-out;
+  ${({ theme }) => theme.breakpoints.down('sm')} {
```
Comment (mattcasey) [inline components/proposals/components/ProposalProperties/components/ProposalRubricCriteriaInput.tsx:36]: can u make both rules mobile-first? i find it confusing to mix

**Review by mattcasey — APPROVED** [review]
minor requests for the css

File: components/proposals/components/ProposalProperties/ProposalProperties.tsx (hunk 1 above)
Comment (valentinludu) [reply components/proposals/components/ProposalProperties/ProposalProperties.tsx:518]: I needed to get rid of width and height. Ofc I could have used !important but I think this is cleaner.

**Pull request merged**